src/Tests/Project/JobRunner.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+
+import {JobRunner} from '../../Project/JobRunner';
+import {WorkJobs} from '../../Project/WorkJobs';
+import {obtainWorkspaceRoot} from '../../Utils/Helpers';
+import {Logger} from '../../Utils/Logger';
+import {MockJob} from '../MockJob';
+
+suite('Project', function() {
+  suite('JobRunner', function() {
+    const logger = new Logger();
+    suite('#start()', function() {
+      test('jobs are done', function(done) {
+        let jobRunner = new JobRunner(logger);
+        const workspaceRoot: string = obtainWorkspaceRoot();
+        let workJobs = new WorkJobs();
+        workJobs.push(new MockJob('mockup'));
+        workJobs.push(new MockJob('mockup'));
+        assert.strictEqual(workJobs.length, 2);
+
+        // overwrite events so that multiple events will be emitted
+        jobRunner.on('cleanup', function() {
```

Comment:
Maybe?
```suggestion
        jobRunner.on(K_CLEANUP, function() {
```

Reply:
`K_CLEANUP` cannot be accessed from the outer. The only `JobRunner` class is exposed.

Reply:
If anyone fixes `K_CLEANUP`, how to do it? Unit tests would not pass so the person should also fix the unit test.

Reply:
@YongseopKim 

> `K_CLEANUP` cannot be accessed from the outer. The only `JobRunner` class is exposed.

Ah, now I've become realized that it was.

> If anyone fixes `K_CLEANUP`, how to do it? Unit tests would not pass so the person should also fix the unit test.

That is what I'm worrying about.

@seanshpark I think `K_` consts need to be exported. What do you think about this?

Reply:
However, the const vars are used only in the JobRunner. To test these, we need them now. Therefore, I don't think the vars should be exported because I commented above.

> Unit tests would not pass so the person should also fix the unit test.